plugins/ltac/tacentries.ml

```diff
@@ -353,7 +353,7 @@ let extend_atomic_tactic name entries =
       let default = epsilon_value inj e in
       match default with
       | None -> raise NonEmptyArgument
-      | Some def -> Tacintern.intern_tactic_or_tacarg Tacintern.fully_empty_glob_sign def
+      | Some def -> Tacintern.intern_tactic_or_tacarg (Genintern.empty_glob_sign Environ.empty_env) def
```

Comment:
I need help to get rid of this one (it was there before, this diff is just making it more obvious). @ppedrot, any suggestion?

Reply:
This code is to be called as a subpart of a toplevel command, so I guess you can just call `Global.env`, or better take it as an argument and get the global environment at the verncular command level.